back-end/src/main/java/com/skynet/skymed/service/EmailDePacienteService.java

```diff
@@ -66,7 +62,7 @@ public void enviaEmail(Pessoa objetoPessoa) throws IOException {
 			Response response = sg.api(request);
 			System.out.println(response.getStatusCode());
```

Comment:
```suggestion
			System.out.println(response.getStatusCode());
```
???? Pra q tu ta printando msg

Reply:
Isso só por que eu tava querendo ver o **retorno do metodo**